apps/extension/src/ui/domains/Account/AccountAdd/AccountAddQr/context.tsx

```diff
@@ -80,7 +78,7 @@ export const reducer = (state: AddQrState, action: Action): AddQrState => {
 
       if (!scanned) return state
       if (!scanned.isAddress)
-        return { type: "SCAN", enable: true, scanError: "QR code is not valid" }
+        return { type: "SCAN", enable: true, scanError: "QR code does not contain an address" }
```

Comment:
Changed this so that in the future it won't be confused with the other `QR code is not valid` error.